http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/LeoMetricsMonitor.scala

```diff
@@ -173,11 +173,12 @@ class LeoMetricsMonitor[F[_]](config: LeoMetricsMonitorConfig,
                 relayPath = Uri
                   .unsafeFromString(baseUri.asString) / s"${app.appName.value}-${app.workspaceId.map(_.value.toString).getOrElse("")}"
                 isUp <- serviceName match {
-                  case ServiceName("cbas")     => cbasDAO.getStatus(relayPath, authHeader).handleError(_ => false)
-                  case ServiceName("cbas-ui")  => cbasUiDAO.getStatus(relayPath, authHeader).handleError(_ => false)
-                  case ServiceName("cromwell") => cromwellDAO.getStatus(relayPath, authHeader).handleError(_ => false)
-                  case ServiceName("wds")      => wdsDAO.getStatus(relayPath, authHeader).handleError(_ => false)
-                  case ServiceName("batch")    => hailBatchDAO.getStatus(relayPath, authHeader).handleError(_ => false)
+                  case ServiceName("cbas")    => cbasDAO.getStatus(relayPath, authHeader).handleError(_ => false)
+                  case ServiceName("cbas-ui") => cbasUiDAO.getStatus(relayPath, authHeader).handleError(_ => false)
+                  case ServiceName("cromwell") | ServiceName("cromwell-reader") =>
```

Comment:
It looked like `cromwell-writer` was a possible servine name too?

Reply:
Yes, I believe that will be taken care of in #3621

Reply:
TOL I wonder whether just using "cromwell" for both makes sense, but not enough to request that in this PR

Reply:
Had the same thought while working through this. I think one benefit I see of having them labeled differently is that it's easier to pick out whether it's a reader or writer from a proxy URL object, since they both will end in `/cromwell` but would have different keys. Definitely open to changing this but I think it might change more than just this PR as well since there are related CBAS changes already in progress.